DigitalLearningSolutions.Data.Migrations/202307101239_AddAcceptableUsePolicyRecord.cs

```diff
@@ -0,0 +1,222 @@
+                                                    <li>any automated analytical technique aimed at analyzing text and data in digital form to generate information which includes but is not limited to patterns, trends, and correlations.</li>
+                                                </ol>
+                                            </li>
+                                            <li>The provisions in this paragraph should be treated as an express reservation of our rights in this regard, including for the purposes of Article 4(3) of Digital Copyright Directive ((EU) 2019/790).</li>
+                                            <li>This paragraph shall not apply insofar as (but only to the extent that) we are unable to exclude or limit text or data mining or web scraping activity by contract under the laws which are applicable to us.</li>
+                                        </ol>
+                                    </li>
+                                    <li class=""h2 nhsuk-heading-l nhsuk-u-padding-bottom-2 nhsuk-u-margin-0"">
+                                        Breach of this Acceptable Use Policy
+                                        <ol class=""h4 nhsuk-body-m nhsuk-u-padding-top-4 nhsuk-u-padding-bottom-2 nhsuk-u-padding-left-0"">
+                                            Failure to comply with this Acceptable Use Policy constitutes a material breach of this Acceptable Use Policy upon which you are permitted to use the Platform and may result in our taking all or any of the following actions:
+                                            <li class=""nhsuk-u-padding-top-2 nhsuk-u-padding-bottom-2"">immediate, temporary, or permanent withdrawal of your right to use the Platform;</li>
+                                            <li>immediate, temporary, or permanent removal of any Contribution uploaded by you to the Platform;</li>
+                                            <li>issue of a warning to you;</li>
+                                            <li>legal proceedings against you for reimbursement of all costs on an indemnity basis (including, but not limited to, reasonable administrative and legal costs) resulting from the breach, and/or further legal action against you;</li>
+                                            <li>disclosure of such information to law enforcement authorities as we reasonably feel is necessary or as required by law; and/or</li>
+                                            <li>any other action we reasonably deem appropriate.</li>
+                                        </ol>
+                                    </li>
+                                </ol>";
+
+            Execute.Sql(@"INSERT INTO [dbo].[Config] ([ConfigName], [ConfigText], [IsHtml])
```

Comment:
Add updated date here to current date.

Reply:
For insert operation on Config table, default values for CreateDate and UpdateDate are current dates. Added both dates explicitly. 
Please review.